docs/install/any-kubernetes-cluster.md

```diff
@@ -23,15 +23,20 @@ showlandingtoc: "false"
 You can install Knative by applying YAML files using the `kubectl` CLI.
 You can install the Serving component, Eventing component, or both on your cluster.
 
+## System Minimum Requirements
+For prototyping purposes, Knative can work on local deployments of Kubernetes (minikube, kind, etc.).
+
+For production purposes, we recommend:
+- If you have only one node in your cluster, you will need at least 6 CPUs, 6 GB of memory, and 30 GB of disk storage.
+- If you have multiple nodes in your cluster, for each node you will need at least 2 CPUs, 4 GB of memory, and 20 GB of disk storage.
```

Comment:
So , I just set up a `kind` cluster with a single node with 2 CPU and 4GB RAM using Kourier (cribbed from https://github.com/csantanapr/knative-kind):

```shell
$ kubectl describe nodes
Name:               kind-control-plane
Roles:              master
...
Capacity:
  cpu:                2
  memory:             4042468Ki
  pods:               110
Allocatable:
  cpu:                2
  ephemeral-storage:  103211296Ki
  memory:             4042468Ki
  pods:               110
Non-terminated Pods:          (23 in total)
  Namespace                   Name                                          CPU Requests  CPU Limits  Memory Requests  Memory Limits  AGE
  ---------                   ----                                          ------------  ----------  ---------------  -------------  ---
  default                     hello-00001-deployment-75c8ff6d45-gs64n       25m (1%)      0 (0%)      0 (0%)           0 (0%)         14s
  knative-eventing            eventing-controller-d666b4657-7jdw6           100m (5%)     0 (0%)      100Mi (2%)       0 (0%)         2m21s
  knative-eventing            eventing-webhook-778b6b8cf4-mg8vn             20m (1%)      200m (10%)  20Mi (0%)        200Mi (5%)     2m21s
  knative-eventing            imc-controller-5f4bdf86cf-gt97m               0 (0%)        0 (0%)      0 (0%)           0 (0%)         111s
  knative-eventing            imc-dispatcher-54bfc97957-rvt2c               0 (0%)        0 (0%)      0 (0%)           0 (0%)         111s
  knative-eventing            mt-broker-controller-ff696f56c-k8zfg          100m (5%)     0 (0%)      100Mi (2%)       0 (0%)         103s
  knative-eventing            mt-broker-filter-d44b776d8-4sq75              100m (5%)     0 (0%)      100Mi (2%)       0 (0%)         103s
  knative-eventing            mt-broker-ingress-6c8487b74c-gh2fm            100m (5%)     0 (0%)      100Mi (2%)       0 (0%)         103s
  knative-serving             3scale-kourier-control-547fc6c565-4rspk       0 (0%)        0 (0%)      0 (0%)           0 (0%)         5m12s
  knative-serving             activator-86956bbd6f-qvrpn                    300m (15%)    1 (50%)     60Mi (1%)        600Mi (15%)    5m23s
  knative-serving             autoscaler-54cbd576f6-vj28h                   30m (1%)      300m (15%)  40Mi (1%)        400Mi (10%)    5m23s
  knative-serving             controller-79c9cccd6f-2rsxp                   100m (5%)     1 (50%)     100Mi (2%)       1000Mi (25%)   5m23s
  knative-serving             webhook-5fd484cf4-xl5c5                       100m (5%)     500m (25%)  100Mi (2%)       500Mi (12%)    5m23s
  kourier-system              3scale-kourier-gateway-5665d74bd4-zvxxb       0 (0%)        0 (0%)      0 (0%)           0 (0%)         5m12s
  kube-system                 coredns-f9fd979d6-g96x6                       100m (5%)     0 (0%)      70Mi (1%)        170Mi (4%)     5m52s
  kube-system                 coredns-f9fd979d6-pfvk7                       100m (5%)     0 (0%)      70Mi (1%)        170Mi (4%)     5m52s
  kube-system                 etcd-kind-control-plane                       0 (0%)        0 (0%)      0 (0%)           0 (0%)         5m51s
  kube-system                 kindnet-6z76p                                 100m (5%)     100m (5%)   50Mi (1%)        50Mi (1%)      5m52s
  kube-system                 kube-apiserver-kind-control-plane             250m (12%)    0 (0%)      0 (0%)           0 (0%)         5m51s
  kube-system                 kube-controller-manager-kind-control-plane    200m (10%)    0 (0%)      0 (0%)           0 (0%)         5m51s
  kube-system                 kube-proxy-d4l7l                              0 (0%)        0 (0%)      0 (0%)           0 (0%)         5m52s
  kube-system                 kube-scheduler-kind-control-plane             100m (5%)     0 (0%)      0 (0%)           0 (0%)         5m51s
  local-path-storage          local-path-provisioner-78776bfc44-r247s       0 (0%)        0 (0%)      0 (0%)           0 (0%)         5m51s
Allocated resources:
  (Total limits may be over 100 percent, i.e., overcommitted.)
  Resource           Requests     Limits
  --------           --------     ------
  cpu                1825m (91%)  3100m (155%)
  memory             910Mi (23%)  3090Mi (78%)
  ephemeral-storage  0 (0%)       0 (0%)
```

CPU gets close at 2 CPUs with one user-container running (requests is near capacity, limits is over), but I think we could say that 3 CPUs and 5GB RAM is a lower bound for reasonable experimentation.

I'll also point out this is with `kind` requesting almost 1 CPU (2*0.1+0.1+0.25+0.2+0.1 = 0.85)